includes/class-wc-data-structurer.php

```diff
@@ -0,0 +1,180 @@
+<?php
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly
+}
+
+/**
+ * Structured data's handler and generator using JSON-LD format.
+ *
+ * @class 		WC_Data_Structurer
+ * @version		2.7.0
+ * @package		WooCommerce/Classes
+ * @category	Class
+ * @author 		Clement Cazaud
+ */
+class WC_Data_Structurer {
```

Comment:
That's what I've done originally but finally I did change it when I realised that eg: `WC_Structured_Data->get_structured_data()` might sound a bit repetitive...